serve/filters/filter/tx.go

```diff
@@ -0,0 +1,101 @@
+package filter
+
+import (
+	"github.com/gnolang/gno/tm2/pkg/bft/types"
+)
+
+type Options struct {
```

Comment:
Here are a couple of ideas for possible options:
- message type
- fee
- memo

For specific message types, we can have different options...

This can be implemented in a future PR, just throwing this out there as a suggestion